frontend/src/app/App.js

```diff
@@ -37,12 +38,15 @@ class App extends Component {
             <>
               <Header />
               <Container>
-                <Route exact path={ROUTES.DASHBOARD} component={Dashboard} />
-                <Route exact path={ROUTES.CREATE_MATCH} component={CreateMatchPage} />
-                <Route exact path={ROUTES.ADD_USER} component={AddUserPage} />
-                <Route exact path={ROUTES.USER_LIST} component={UserListPage} />
-                <Route exact path={ROUTES.MATCH_LIST} component={MatchListPage} />
-                <Route exact path={ROUTES.PROFILE} component={Profile} />
+                <Switch>
+                  <Route exact path={ROUTES.LEADERBOARDS} component={LeaderboardsPage} />
+                  <Route exact path={ROUTES.CREATE_MATCH} component={CreateMatchPage} />
+                  <Route exact path={ROUTES.ADD_USER} component={AddUserPage} />
+                  <Route exact path={ROUTES.USER_LIST} component={UserListPage} />
+                  <Route exact path={ROUTES.MATCH_LIST} component={MatchListPage} />
+                  <Route exact path={ROUTES.PROFILE} component={Profile} />
+                  <Route exact path={ROUTES.DASHBOARD} component={Dashboard} />
```

Comment:
now add `<Route component={Dashboard} />`
on the last line. that would enforce that any invalid url redurects to the dashborad, which is not the case now

Reply:
done [here](https://github.com/salsita/foosball-rating/pull/92/commits/d7c17da31ec421acbb90f4a19ce22c34412cdf35#diff-ec0334f02e0502ad4cd3b898702ec2e2)